code/cutscene/cutscenes.cpp

```diff
@@ -88,31 +152,29 @@ void cutscene_init()
 				stuff_int(&junk);
 			}
 
-			cutinfo.flags.reset();
+			csnp->flags.reset();
```

Comment:
This will clear the flags for any modular table entry that modifies an existing entry, even if the new modular table entry doesn't modify flags.  To avoid this, you will need to enclose this in a check with a `first_time` variable.  This variable should default to false but be set to true after the line `Cutscenes.push_back(csnt);`